doc/source/composites.rst

```diff
@@ -347,6 +347,12 @@ Here we see two changes:
 The modifier above is a built-in that normalizes the Solar zenith
 angle to Sun being directly at the zenith.
 
+More examples can be found in Satpy source code directory
+``satpy/etc/composite.yaml``.
```

Comment:
Could/should this maybe be a link to the actual directory? As is `composite.yaml` isn't a file or directory in the repository.

Reply:
Oops, of course this should be `satpy/etc/composites` and not `composite.yaml`.  As for link to the actual directory, you mean on GitHub?

Reply:
Yeah, link to the directory on GitHub.

Reply:
Fixed.